action.yml

```diff
@@ -9,30 +9,56 @@ inputs:
     required: true
     description: "which action to perform. Valid values are: 'storePlan', 'getPlan', 'taintPlan'"
     default: "storePlan"
-  planPath:
-    description: "path to the Terraform plan file. Required for 'storePlan' and 'getPlan' actions"
-  commitSHA:
-    description: "Commit SHA to use for fetching plan"
+  blobAccountName:
+    description: "the name of the Azure Blob Storage account to store the plan file"
+    required: false
+  blobContainerName:
+    description: "the name of the Azure Blob Storage container to store the plan file"
     required: false
-    default: ''
-  tableName:
-    description: "the name of the dynamodb table to store metadata"
-    required: true
-    default: "terraform-plan-storage"
   bucketName:
     description: "the name of the S3 bucket to store the plan file"
     required: true
```

Comment:
bucket wouldnt be required for Azure either

Reply:
Good catch...although because there's a default value, this doesn't cause issues. I'll update it though, so the docs are right.